tasks/tinc_configure.yml

```diff
@@ -103,3 +125,25 @@
   systemd:
     name: "{{ tinc_service_name }}"
     enabled: yes
+  when: ansible_service_mgr == "systemd"
+
+# this enables the use of the tinc start script provided by OpenWrt
+- name: OpenWRT Configure tinc with UCI
+  template:
+    src: openwrt-tinc-uci-config.j2
+    dest: /etc/config/tinc
+    owner: root
+    group: root
+    mode: 0644
+  when: ansible_os_family == "OpenWrt"
+  tags: uci
+  notify:
+    - Restart Service
+
+- name: OpenWrt ensure tinc is enabled in init and started
+  service:
+    name: "{{ tinc_service_name }}"
+    enabled: yes
+    state: restarted
+  when: ansible_service_mgr == "openwrt_init"
+  tags: start
```

Comment:
could you provide 'start' tag as a list item instead, please?

Reply:
yes - updated to match main.yml style